guacml/splitters/date_splitter.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+
+class DateSplitter:
+    def __init__(self, config):
+        self.config = config
+
+    def holdout_split(self, input):
+        series = input[self.config['date_split_col']]
+        split_point = series.quantile(1 - self.config['holdout_size'], interpolation='lower')
+        return input[series <= split_point], input[series > split_point]
+
+    def cv_splits(self, input):
```

Comment:
I think in the future this needs to be more configurable. Usually you know beforehand how far in the future you want to predict. We should use that length as length for the backtesting folds.